packages/patterns-react/package.json

```diff
@@ -49,8 +49,10 @@
     "@babel/runtime": "^7.5.5",
     "@carbon/ibmdotcom-services": "0.4.0",
     "@carbon/ibmdotcom-styles": "0.4.0",
-    "@carbon/ibmdotcom-utilities": "0.6.0-alpha.511",
+    "@carbon/ibmdotcom-utilities": "^1.1.0-alpha.610",
     "classnames": "2.2.6",
+    "dompurify": "^2.0.6",
```

Comment:
what is `dompurify`?

Reply:
Ah, I need to remove it, installed when I test to handle html in pattern as Drupal generates raw html for content field.

Reply:
ok thanks, also, is `jsdom` being used? don't think we need that either.

Reply:
right, we don't need it. I will remove it too.

Reply:
removed packages with [this commit](https://github.com/carbon-design-system/ibm-dotcom-library/pull/561/commits/00685db4216a15750a44b551f95c0740c848f8ad)